.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [^10, ^12]
+        node-version: [^10.13, ^12.13, ^14.15]
```

Comment:
@poveden Eventually fixed the failing CI by matching here the expectations of supported LTS versions in package.json `engines` field